tests/framework/layer/layer_util.cpp

```diff
@@ -0,0 +1,151 @@
+/*
```

Comment:
This file looks like the one that is used to do a bunch of common tasks in the validation layer. I fail to see how it's relevant here, and it only appears to be used by the existence layer, who's purpose I also don't understand. I would suspect this should be removed.

Reply:
This is from trying to get layer tests working, will remove it.